src/CodeGen_ARM.cpp

```diff
@@ -1100,6 +1256,10 @@ string CodeGen_ARM::mattrs() const {
             arch_flags = "+sve";
         }
 
+        if (target.has_feature(Target::ARMDotProd)) {
```

Comment:
Does this need to be gated by LLVM_VERSION?

Reply:
llvm 8 has it, so we're good